src/app/configs/config-detail/config-detail.component.html

```diff
@@ -8,6 +8,12 @@
             <mat-card-title></mat-card-title>
             <mat-card-subtitle></mat-card-subtitle>
         </mat-card-header>
+        <mat-list>
+            <mat-list-item *ngFor="let item of errorMessages">
+                <mat-icon mat-list-icon>error</mat-icon>
```

Comment:
mat-icon should support color=warn -- which will use the correct color in the color palette.  ex:
`<mat-icon color="warn" mat-list-icon>`